platform-emf/src/main/java/com/softicar/platform/emf/form/popup/EmfFormPopup.java

```diff
@@ -57,6 +57,9 @@ public R getTableRow() {
 		return form.getTableRow();
 	}
 
+	/**
+	 * Delegates to {@link EmfForm#setCallbackAfterCreation(Consumer)}.
```

Comment:
Why would we explicitly describe this - while we don't do the same for the method below? (`addAdditionalValidator`)

Reply:
i think it's necessary to point the dev to the Javadoc, i did the same for the next 2 methods